drivers/serial/uart_cc13xx_cc26xx.c

```diff
@@ -53,6 +65,12 @@ static int uart_cc13xx_cc26xx_poll_in(struct device *dev, unsigned char *c)
 static void uart_cc13xx_cc26xx_poll_out(struct device *dev, unsigned char c)
 {
 	UARTCharPut(get_dev_conf(dev)->regs, c);
+	/*
```

Comment:
Would it make sense to have this be conditional on power management?  Otherwise waiting for transmission to complete is a behavioral change that may impact other things the application would normally be doing after this function.